pkgs/data/fonts/iosevka/default.nix

```diff
@@ -50,19 +49,16 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     nodejs
     nodeIosevka
-    remarshal
     ttfautohint-nox
   ];
 
-  privateBuildPlanJSON =
-    builtins.toJSON { buildPlans.${pname} = privateBuildPlan; };
-  inherit extraParameters;
-  passAsFile = [ "privateBuildPlanJSON" "extraParameters" ];
+  inherit privateBuildPlan extraParameters;
+  passAsFile = [ "privateBuildPlan" "extraParameters" ];
 
   configurePhase = ''
     runHook preConfigure
     ${lib.optionalString (privateBuildPlan != null) ''
-      remarshal -i "$privateBuildPlanJSONPath" -o private-build-plans.toml -if json -of toml
+      cat "$privateBuildPlanPath" > private-build-plans.toml
```

Comment:
Why not `cp`?

Reply:
No idea, to be honest. Should be an easy fix!